.circleci/src/workflows/site-factory.yml

```diff
@@ -9,7 +9,7 @@ jobs:
           only: /.*/
   # Check that circleci configuration is updated
   - check-configuration:
-      ci_update_options: ""
+      ci_update_options: "--ignore-changelog"
```

Comment:
You should add a "Fixed" item to the changelog:
- Fix javascript build which was forgotten in Dockerfile and missing from image

Reply:
This was symptomatic to another issue. Since `site-factory.yml` is autogenerated, it should not be commited otherwise check_configuration job always failed...